hudi-client/hudi-java-client/src/test/java/org/apache/hudi/common/table/read/HoodieFileGroupReaderOnJavaTestBase.java

```diff
@@ -89,6 +90,7 @@ public void commitToTable(List<HoodieRecord> recordList, String operation, boole
             .setRecordKeyFields(writeConfigs.get(KeyGeneratorOptions.RECORDKEY_FIELD_NAME.key()))
             .setPartitionFields(writeConfigs.get(KeyGeneratorOptions.PARTITIONPATH_FIELD_NAME.key()))
             .setPreCombineFields(writeConfigs.get("hoodie.datasource.write.precombine.field"))
+            .setBaseFileFormat(writeConfigs.get(HoodieTableConfig.BASE_FILE_FORMAT.key()))
```

Comment:
hmm, a write config fetch valus from a table config option this is weird.

Reply:
This is how all the parameters are setup in this test.